### DISCUSSION TITLE
百科全书使用问题

### DISCUSSION THREAD

**Discussion by Geeeety** [zotero-chinese/wiki#142] [求助 / Q&A] [answered]
最近在认真看百科全书，但是看完快速开始之后就没法继续看了。因为后续部分都需要展开，而我不知道为什么鼠标点了一直展不开，刷新页面也没用（哭
![Snipaste_2024-03-08_01-05-02](https://github.com/zotero-chinese/wiki/assets/150330120/392d793d-6804-49b4-aa1d-45c13f1f4d86)
）

**Comment by northword** [comment, marked as answer]
抱歉造成困扰，是网站更新导致的bug，已临时修复。

**Reply by Geeeety** [reply]
谢谢你！
